pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -351,6 +365,14 @@ func (c *EtcdProxyController) syncHandler(key string) error {
 		c.recorder.Event(etcdstorage, corev1.EventTypeWarning, ResourceReclaimed, msg)
 	}
 
+	// Once EtcdStorage is successfully deployed, deploy the certificates in the API server namespace.
+	if err = c.updateAPIServerServingCAConfigMaps(etcdstorage); err != nil {
+		return err
```

Comment:
didn't @deads2k have the comment here to continue with `updateAPIServerClientCertSecrets` in case of error in `updateAPIServerServingCAConfigMaps`?